PROJECT_LIFECYCLE.md

```diff
@@ -0,0 +1,121 @@
+# Project Lifecycle
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Projects](#projects)
+- [Project Lifecycle Process](#project-lifecycle-process)
+- [Project Lifecycle Stages](#project-lifecycle-stages)
+- [References](#references)
+
+## Overview
+
+Eiffel Community is home to various projects that enable the members of the community to use, contribute,
+and collaborate on Eiffel.
+
+This document describes how Eiffel Community defines the projects, how the projects can be created, maintained,
+and archived. In addition to the project lifecyle process, stages which the projects can be admitted or transitioned
+into as well as criteria and expectations for the stages are described.
+
+## Projects
+
+The activities of the Eiffel Community are organized into projects targeting different areas within the scope of
+Eiffel Community including but not limited to development and maintenance of Eiffel Protocol, development of new
+functionality that adheres to the protocol, integration of other technologies, testing, documentation, as well as
+the support functions to create and maintain the infrastructure and the toolchain community uses.
+
+## Project Lifecycle Process
+
+New projects must be formally proposed on GitHub on [Eiffel Community](https://github.com/eiffel-community/community)
+repository by creating an issue. The proposals submitted should provide the following information to the best of
+their ability.
+
+- name of the project
+- project description (what it does, why it is valuable, origin and history)
+- project license
+- names of initial maintainers
+- infrastructure needs such as Git repositories
+
+Projects that are already hosted within Eiffel Community but willing to move to a different stage are also
+expected to follow same process as project creation by formally submitting their request on GitHub Eiffel
+Community repository by creating an issue.
+
+### Project Acceptance Process
+
+Project review and acceptance happen directly on the issue asking for project creation. If community members
+require further discussions, ad-hoc meetings can be scheduled to discuss the proposal further but this is not
+mandatory.
+
+Simple majority of Eiffel Community Maintainers is required for a project to be accepted to Eiffel Community.
+
```

Comment:
Add that a PR updating PROJECTS.md will be done within the acceptance process as well (?).

Reply:
done